src/ArcGISEarth.AutoAPI.Examples/ArcGISEarth.AutoAPI.Examples.csproj

```diff
@@ -104,6 +143,19 @@
     <Resource Include="examples.txt" />
   </ItemGroup>
   <Import Project="..\ArcGISEarth.AutoAPI.Shared\ArcGISEarth.AutoAPI.Shared.projitems" Label="Shared" />
+  <ItemGroup>
+    <Resource Include="Icons\send_icon_h.png">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Resource>
+    <Resource Include="Icons\send_icon_n.png">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Resource>
+  </ItemGroup>
+  <ItemGroup>
+    <Resource Include="Icons\API.ico">
```

Comment:
Suggest all lowercase . It's better to rename png files to `send_hover.png` and `send_normal.png`